pom.xml

```diff
@@ -66,6 +66,55 @@
 	</dependencies>
 
 	<build>
+	
+	    <pluginManagement>
+          <plugins>
+            <plugin>
+			  <groupId>com.diffplug.spotless</groupId>
+			  <artifactId>spotless-maven-plugin</artifactId>
+			  <version>2.6.1</version>
+			  <configuration>
+			    <!-- optional: limit format enforcement to just the files changed by this feature branch -->
+			    <ratchetFrom>origin/master</ratchetFrom>
+			    <formats>
+			      <!-- you can define as many formats as you want, each is independent -->
+			      <format>
+			        <!-- define the files to apply to -->
+			        <includes>
+			          <include>*.md</include>
+			          <include>.gitignore</include>
+			        </includes>
+			        <!-- define the steps to apply to those files -->
+			        <trimTrailingWhitespace/>
+			        <endWithNewline/>
+			        <indent>
+			          <tabs>true</tabs>
+			          <spacesPerTab>4</spacesPerTab>
+			        </indent>
+			      </format>
+			    </formats>
+			    <!-- define a language-specific format -->
+			    <java>
+			      <!-- no need to specify files, inferred automatically, but you can if you want -->
+			
+			      <!-- apply a specific flavor of google-java-format -->
+			      <googleJavaFormat>
+			        <version>1.8</version>
+			        <style>AOSP</style>
+			      </googleJavaFormat>
+			
+			      <!-- make sure every file has the following copyright header.
+			        optionally, Spotless can set copyright years by digging
+			        through git history (see "license" section below) -->
+			      <licenseHeader>
+			        <content>/* (C)$YEAR */</content>  <!-- or <file>${basedir}/license-header</file> -->
```

Comment:
good idea to add spotless to the pom, however the correct licenceHeader must be added here otherwise a wrong header is applied to sources. Can you add the correct header here and re-apply spotless?